CONTRIBUTING.md

```diff
@@ -168,6 +168,12 @@ tox -e py38 -- test/cli
 tox -e py38 -- test/cli/commands_test.py
 ```
 
+Running your code changes against a SQL file you can run
+```shell
+sqlfluff parse my-sql.sql
```

Comment:
```suggestion
sqlfluff parse test.sql
```
use `test.sql` for random tests. It's already in the `.gitignore` so people won't accidently commit them.